modules/swagger-codegen/src/main/resources/typescript-angular/modelEnum.mustache

```diff
@@ -1,7 +1 @@
-export enum {{classname}} {
-{{#allowableValues}}
-{{#enumVars}}
-    {{{name}}} = <any> {{{value}}}{{^-last}},{{/-last}}
-{{/enumVars}}
-{{/allowableValues}}
-}
+export type {{classname}} = {{#allowableValues}}{{#enumVars}}{{{value}}}{{^-last}} | {{/-last}}{{/enumVars}}{{/allowableValues}};
```

Comment:
Please don't remove the newline in the end. (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)